internal/proto/controller/storage/servers/store/v1/worker_auth.proto

```diff
@@ -40,14 +40,14 @@ message WorkerAuth {
 // WorkerCertBundle contains all fields related to a WorkerCertBundle resource
 message WorkerCertBundle {
   // The public key of the issuing root certificate
```

Comment:
This comment is a bit confusing -- if it's actually part of the root certificate, I would suggest naming it something like `ca_certificate_public_key` or `root_certificate_public_key`...as is it seems like it's talking about the client certificate since that's the worker cert.

Reply:
Good catch, I'll rename it